tests/conftest.py

```diff
@@ -220,7 +221,21 @@ def application_process(
 
 @fixture(scope="session")
 def docker_client() -> DockerClient:
-    return docker.from_env()
+    tests_id = secrets.token_hex(5)
+    docker_client = docker.from_env()
+    setattr(docker_client, "test_id", tests_id)
+    yield docker_client
+
+    exited_ids = []  # type: List[str]
```

Comment:
```suggestion
    exited_ids: Listr[str] = []
```
If explicit typing is needed, use Python 3 types.